src/hatti/shared.cljs

```diff
@@ -35,7 +35,9 @@
   "An initial, empty, app-state, which can be modified to change dataviews."
   []
   (atom
-   {:views {:all [:overview :map :table :chart :saved-charts :settings]
+   {:views {:all [:overview :map :table :chart :saved-charts :settings
+                  :map-table
```

Comment:
we talked about this on slack, right now Secretary will only be able to redirect to routes that are added here, so we can't actually remove them. Before merging to master we'll need a solution for this.